tests/integration/src/rust_masm_tests/rust_sdk.rs

```diff
@@ -127,6 +127,7 @@ fn rust_sdk_p2id_note_script() {
 }
 
 #[test]
+#[ignore = "until undefined module 'intrinsics::mem' issue is fixed"]
```

Comment:
Should we create an issue for this?

Reply:
I'm looking into both this and the other issue today, and once I know more about where the error comes from, I'll open an issue for it. Right now it's unclear if it's a bug, or something actually missing somewhere.